### PR TITLE
atomic history update

### PR DESCRIPTION
**Pull request by briangough**
this is a minimal change to move the uncompressed history ops into the redis multi for the document

### CODE REVIEW

File: app/coffee/HistoryRedisManager.coffee [hunk 1]
```diff
@@ -8,13 +8,9 @@ module.exports = HistoryRedisManager =
 	pushUncompressedHistoryOps: (project_id, doc_id, ops = [], callback = (error, length) ->) ->
```
Comment (jpallen) [inline app/coffee/HistoryRedisManager.coffee]: This is now incorrectly named. Something like `recordDocHasHistoryOps`?

File: app/coffee/HistoryRedisManager.coffee [hunk 2]
```diff
@@ -8,13 +8,9 @@ module.exports = HistoryRedisManager =
...
-			(cb) -> rclient.rpush Keys.uncompressedHistoryOps({doc_id}), jsonOps..., cb
-			(cb) -> rclient.sadd Keys.docsWithHistoryOps({project_id}), doc_id, cb
-		], (error, results) ->
+		logger.log project_id: project_id, doc_id: doc_id, "marking doc in project for history ops"
+		rclient.sadd Keys.docsWithHistoryOps({project_id}), doc_id, (error) ->
 			return callback(error) if error?
-			[length, _] = results
-			callback(error, length)
\ No newline at end of file
+			rclient.llen Keys.uncompressedHistoryOps({doc_id}), (error, length) ->
```
Comment (jpallen) [inline app/coffee/HistoryRedisManager.coffee]: We can get this length from the call to `updateDocument` (with appropriate plumbing), so there shouldn't be a need for this extra redis call

**Comment by briangough** [comment]
These changes get the length from the redis update directly now.

File: app/coffee/HistoryManager.coffee [hunk 3]
```diff
@@ -21,16 +21,17 @@ module.exports = HistoryManager =
 				error = new Error("track changes api returned a failure status code: #{res.statusCode}")
 				return callback(error)
 
-	FLUSH_EVERY_N_OPS: 50
-	pushUncompressedHistoryOps: (project_id, doc_id, ops = [], callback = (error) ->) ->
+	FLUSH_EVERY_N_OPS: 100
+	pushUncompressedHistoryOps: (project_id, doc_id, ops = [], length, callback = (error) ->) ->
```
Comment (jpallen) [inline app/coffee/HistoryManager.coffee]: This method is also wrongly named now. Not sure what it should be though. `recordAndFlushHistoryOps`?

**Pull request merged**